__tests__/endpoint.tests/endpoint.test.js

```diff
@@ -116,6 +116,48 @@ describe("articles", () => {
     });
   });
 
+  describe("GET all articles with queries", () => {
+    test("GET: 200 articles are sorted by created_at desc by default", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeSortedBy("created_at", {
+            descending: true,
+          });
+        });
+    });
+
+    test("GET:200 articles are sorted by valid property and valid order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=author&order=asc")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeSortedBy("author", {
+            ascending: true,
```

Comment:
toBeSortedBy doesnt have an ascending property but it does default to descending: false so thats how this is working. I would change it to descending: false to go hand in hand with this